### PR TITLE
Update changelog SKIP_USERS list

### PR DESCRIPTION
**Pull request by dchyun**
### :pushpin: Summary

Updated the `SKIP_USERS` list for the changelog to add in the new username for Dylan Hyun.

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

**Review by didoo — APPROVED** [review]
👍 🎉 🚀

**Comment by KristinLBradley** [comment]
@dchyun This PR is fine to merge whenever you are ready.

(For small, low risk PRs like this it's normally fine to merge right after receiving at least one approval. For larger, more complex PRs though you may want to wait for review and approval of multiple relevant parties.)

**Pull request merged**
